drive/types.bal

```diff
@@ -237,31 +237,31 @@ public type GeneratedFileMetadata record {
     ContentRestrictions contentRestrictions?;
 };
  
-# Record Type to accpet string values  
+# Record Type to accpet string values.  
 public type StringKeyValuePairs record {|
     string...;
 |};
 
-# Record Type to accept string[] values
+# Record Type to accept string[] values.
 public type StringArrayValuePairs record {|
     string[]...;
 |};
 
-# Record Type to accept float values  
+# Record Type to accept float values.
 public type StorageQuota record {|
     float...;
 |};
 
 # Restrictions for accessing the content of the file. Only populated if such a restriction exists.
 #
 # + reason - Reason for why the content of the file is restricted. This is only mutable on requests that also 
-#            set readOnly=true.  
+#            set readOnly=true
 # + readOnly - Whether the content of the file is read-only. If a file is read-only, a new revision of the file may not 
-#              be added,comments may not be added or modified, and the title of the file may not be modified.  
-# + restrictionTime - The time at which the content restriction was set (formatted RFC 3339 timestamp). 
-#                     Only populated if readOnly is true.  
-# + type - The type of the content restriction. Currently the only possible value is globalContentRestriction.  
-# + restrictingUser - The user who set the content restriction. Only populated if readOnly is true.  
+#              be added,comments may not be added or modified, and the title of the file may not be modified
+# + restrictionTime - The time at which the content restriction was set (formatted RFC 3339 timestamp)
```

Comment:
Suggestion: "The time (i.e., formatted RFC 3339 timestamp) st which the content restriction was set. This is displayed only if the `readOnly` parameter is set to `true`.

Reply:
Hi,
We have used the same content from [1] for parameter descriptions
[1]https://developers.google.com/drive/api/v3/reference/files
Thanks